CMakeLists.txt

```diff
@@ -296,7 +296,7 @@ add_library(PKWare STATIC
   3rdParty/PKWare/implode.cpp)
 target_include_directories(PKWare PUBLIC 3rdParty/PKWare)
 
-set(devilutionx_SRCS
+set(devilutionxlib_SRCS

```

Comment:
Please use `libdevilutionx` for the library target as it is more conventional

Reply:
Done 🙂